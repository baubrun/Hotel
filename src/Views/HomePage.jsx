import React from "react";
import HeroImg from "../components/HeroImg";
import Banner from "../components/Banner";
import { Link } from "react-router-dom";
import Amenities from "../components/Amenities"
import FeaturedRoomsContainer from "../components/FeaturedRoomsContainer"


const HomePage = () => {
  return (
    <>
    <HeroImg hero="defaultHero">
      <Banner title="Luxury Rooms" subtitle="Deluxe Rooms starting at $299">
        <Link to="/rooms" className="btn-primary">
          View Rooms
        </Link>
      </Banner>
    </HeroImg>
    <Amenities />
    <FeaturedRoomsContainer />
    </>
  );
};

export default HomePage;
