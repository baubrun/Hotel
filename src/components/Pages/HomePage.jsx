import React from "react";
import HeroImg from "../HeroImg";
import Banner from "../Banner";
import { Link } from "react-router-dom";
import Amenities from "../Amenities"
import FeaturedRoomsContainer from "../FeaturedRoomsContainer"


const HomePage = () => {
  return (
    <>
    <HeroImg>
      <Banner title="Luxury Rooms" subtitle="Deluxe Rooms starting at $299">
        <Link to="/rooms" className="btn-primary">
          View Rooms
        </Link>
      </Banner>
    </HeroImg>
    <Amenities />
    <FeaturedRoomsContainer />
    </>
  );
};

export default HomePage;
