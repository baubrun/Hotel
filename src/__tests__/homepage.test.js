import React from "react";
import {
    shallow,
} from "enzyme";
import HomePage from "../Views/HomePage"
import toJSON from "enzyme-to-json"

describe("Homepage", () => {
    const wrapper = shallow(<HomePage />)

    it('renders correctly', () => {
        expect(toJSON(wrapper)).toMatchSnapshot()
    })

});


