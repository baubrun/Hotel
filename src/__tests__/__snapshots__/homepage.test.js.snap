// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<Fragment>
  <HeroImg
    hero="defaultHero"
  >
    <Banner
      subtitle="Deluxe Rooms starting at $299"
      title="Luxury Rooms"
    >
      <Link
        className="btn-primary"
        to="/rooms"
      >
        View Rooms
      </Link>
    </Banner>
  </HeroImg>
  <Amenities />
  <FeaturedRoomsContainer />
</Fragment>
`;
